code/modules/mob/living/carbon/xenomorph/say.dm

```diff
@@ -18,6 +18,12 @@
 /mob/living/carbon/xenomorph/shrike/hivemind_end()
 	return "</span>"
 
+/mob/living/carbon/xenomorph/hivemind/hivemind_start()
+	return "<span class='game say hivemind xenoshrike'>The Hivemind"
```

Comment:
```suggestion
	return "<span class='game say hivemind'>The Hivemind"
```

Update the css to add the font size directly.
`.hivemind				{color: #c586c0; font-size: 1.1em;}` could work.